python/lsst/ts/tunablelaser/config_schema.py

```diff
@@ -46,7 +46,7 @@
     type: number
   optical_configuration:
     description: The mirror alignment configuration for the laser
-    enum: ["SCU","No SCU","F1 SCU","F1 No SCU","F2 SCU","F2 No SCU"]
+    enum: ["SCU","F1 SCU","F2 SCU","No SCU","F1 No SCU","F2 No SCU"]
```

Comment:
Consider making this dynamic by using the enumeration to generate the valid entries:, e.g.;

```
CONFIG_SCHEMA = yaml.safe_load(
    f"""
    ...
    enum: {[e.value for e in OpticalConfiguration]}
    ...
    """
```

Reply:
In order to do this, I'd have to import OpticalConfiguration. Does that work in config_schema?